pkg/kubelet/cpumanager/policy.go

```diff
@@ -24,6 +24,7 @@ import (
 // Policy interface used by cpu manager, shall implement logic for cpu to pod
 // assignment
 type Policy interface {
+	IsUnderPressure() bool
```

Comment:
Oh! Ok, I see that state _and_ policy as accessible from manager.  I thought state was only accessible by policy.  I still think pressure is a policy concept, not a state one.